test/unit/PanoImageRenderer/ImageLoader.spec.js

```diff
@@ -217,6 +252,82 @@ describe("ImageLoader", function() {
 		})
 	});
 
+	describe("#onceLoaded", function() {
+		it("should work with single image", (done) => {
+			// Given
+			const inst = new ImageLoader();
+			const image = new Image();
+			image.src = "./images/PanoViewer/only-used-for-destroy-test.png";
```

Comment:
@happyhj This image (`only-used-for-destroy-test.png`) is intended for not cached image. It's no more useless this image but it looks better to use other image for testing and letting it be used in other cache test.